guides-markdown/multisig-cli.md

```diff
@@ -0,0 +1,463 @@
+[many options available](https://bcoin.io/api-docs/#create-a-wallet)
+for wallet creation, but all Alice needs to do is set up her wallet as a 3-of-5.
+We'll also specify Segregated Witness, although that option will be a bcoin default
+[very soon.](https://github.com/bcoin-org/bcoin/pull/827)
+
+```
+$ bwallet-cli mkwallet --id=Alice --witness=true --m=3 --n=5
+```
+
+Right away, if Alice checks her wallet's default account, she'll notice it has not
+been initialized yet. That's because it is waiting for more multisig keys:
+
+```
+bwallet-cli --id=Alice account get default
+
+{
+  "name": "default",
+  "initialized": false,
+  "witness": true,
+  "watchOnly": false,
+  "type": "multisig",
+  "m": 3,
+  "n": 5,
+  "accountIndex": 0,
+  "receiveDepth": 0,
+  "changeDepth": 0,
+  "nestedDepth": 0,
+  "lookahead": 10,
+  "receiveAddress": null,
+  "changeAddress": null,
+  "nestedAddress": null,
+  "accountKey": "tpubDDXZKjYarBCHxq9GHQHfRjRVQm8DuVAu33LoigMfHJri1ynqsWVBQbMWQbFU5nT3vabniRrtgJtaDEqnpB9GKQTyac5SbUw4589SvwrYKqQ",
+  "keys": [],
+  "balance": {
+    "tx": 0,
+    "coin": 0,
+    "unconfirmed": 0,
+    "confirmed": 0
+  }
+}
+```
+
+Notice how she DOES have an `accountKey` (or "xpub" -- on testnet it's a `tpub`).
+When bcoin created this wallet, it generated entropy and created a
+[BIP44 master private key](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki)
+for Alice. That will be the key she uses to sign the multisig transaction if she approves.
+
+We can repeat the process for the four other friends:
+
+```
+$ bwallet-cli mkwallet --id=Bob --witness=true --m=3 --n=5
+$ bwallet-cli mkwallet --id=Charlie --witness=true  --m=3 --n=5
+$ bwallet-cli mkwallet --id=David --witness=true  --m=3 --n=5
+$ bwallet-cli mkwallet --id=Erica --witness=true  --m=3 --n=5
+```
+
+After generating each wallet we can get the `accountKey` from each wallet's default account:
+
+
+```
+$ bwallet-cli --id=Alice account get default | jq .accountKey
+"tpubDDXZKjYarBCHxq9GHQHfRjRVQm8DuVAu33LoigMfHJri1ynqsWVBQbMWQbFU5nT3vabniRrtgJtaDEqnpB9GKQTyac5SbUw4589SvwrYKqQ"
+$ bwallet-cli --id=Bob account get default | jq .accountKey
+"tpubDDpd2qFrbXW35YkebSqvihtEsUW9ynmpy61FeVmquPzYzYmFupmabBnHwhJjJighrt2Cmss2Ndg7SJ1iXLuQvZia8KCRWvPTBEacpx6BARf"
+$ bwallet-cli --id=Charlie account get default | jq .accountKey
+"tpubDDtiGfAVwUAz2wHGh24TCsk6smz8QbHLqr7swEjS8YiEm6BczQPjqRVBuEqPefUCLur1U2VB4P8TUXQg9vXqzQdUx8bynP574sbzuADmRff"
+$ bwallet-cli --id=David account get default | jq .accountKey
+"tpubDDGK1KoY3YQ1xSwz4AyigwWPickr4mFVtRxu7ZZMQz5BDFGNF6SkgJq2kEfTzuudZVgy84rMVZeMv8GDVod2AGxgZSy3fBpWHpYHTEdUkZc"
+$ bwallet-cli --id=Erica account get default | jq .accountKey
+"tpubDCGKejeBy1oMoZ57NvG2u1X7pffLHkKxew1RAohEzaoPxSSXPdy7st8caWdwymqStLSXxsC4cvKcm9FSLS2PNEcS5rFHo5ccL9xCecqkQth"
+```
+
+Now each friend needs to import everyone else's public keys into their own wallet.
+This will be a bit tedious for us because we have to do it for everybody but
```

Comment:
`This will be a bit tedious`...

```bash
bwallet-cli --id=Bob account get default \
    | jq -r .accountKey \
    | xargs bwallet-cli --id=Alice --account=default shared add
```

Its also possible to wrap this whole thing in a loop for each cosigner, `--id=$COSIGNER` for the get command. Would make a copy/paste easier potentially but also makes it more complex

Reply:
Hmmm, I think this creeps the scope a bit, I really want super simple commands only. The only reason I used `jq` above is to clean the output. I really want readers to see each step as verbose as possible.